.github/workflows/publish.yml

```
@@ -0,0 +1,54 @@
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+name: Deploy Extension
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3.0.2
```

```suggestion
        uses: actions/checkout@v3
```

doesn't v3 get latest from that series?